apex/transformer/testing/distributed_test_base.py

```diff
@@ -1,20 +1,28 @@
+import os
 import sys
+import unittest
 
 import torch
 from torch import distributed as dist
 from torch.testing._internal import common_utils
 from torch.testing._internal import common_distributed
 
+HAS_TORCH_UCC = None
+try:
+    import torch_ucc
+    HAS_TORCH_UCC = True
+except ImportError:
+    HAS_TORCH_UCC = False
 
-class DistributedTestBase(common_distributed.MultiProcessTestCase):
 
-    BACKEND_NCCL = "nccl"
+class DistributedTestBase(common_distributed.MultiProcessTestCase):
 
     def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
 
     def setUp(self) -> None:
         super().setUp()
+        self._setup_pre_spawn()
```

Comment:
Is this needed if the method is currently just `pass`?

Reply:
`UccDistributedTestBase` uses this see around L.88

Reply:
Thanks, my bad, thought this was for the NCCL test subclass